### ISSUE TITLE
Blank files page on fresh install

### ISSUE THREAD

**Issue by desultory** [owncloud/core#29589]
### Steps to reproduce
1. Install owncloud
2. Attempt to view files

### Expected behaviour
Tell us what should happen
I am expecting to see all of my files listed on the files page when I log in, I can't even see the default files.  I used the owncloud app on my android phone to upload a test file and it uploaded properly and is in the folder containing the default files but is not visible on the web interface/

### Actual behaviour
Files page is blank https://i.imgur.com/0WC275q.png

### Server configuration
**Operating system**: Debian 9

**Web server:** Apache 2.4

**Database:** mysql  Ver 15.1 Distrib 10.1.26-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**PHP version:** 7.0

**ownCloud version:** (see ownCloud admin page) 10.0.3.3

**Updated from an older ownCloud or fresh install:** fresh install

**Where did you install ownCloud from:** Owncloud website

**Signing status (ownCloud 9.0 and above):** default?

```
Integrity/failed report
No errors have been found.
```

**The content of config/config.php:**

```
https://gist.github.com/desultory/4ddac435cea2c4083ead45b262f4e55d
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
Firefox / owncloud app for android

**Operating system:**
widows 10 / android 7
#### Web server error log
```
apache is throwing no errors
```

#### ownCloud log (data/owncloud.log)
```
https://gist.github.com/desultory/685dbcb5d9085dcc4831bbd4f3ed10e8
```

#### Browser log
```
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src my_site”). Source: (function(a,p,g,C){var c={safeWindow:{},....
files:1
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src m_site 'unsafe-eval'”). Source: (function(a,p,g,C){var c={safeWindow:{},....
files:1
JQMIGRATE: Migrate is installed, version 1.4.0
jquery-migrate.min.js:2:542
Error: call to Function() blocked by CSP
handlebars.js:3562:15
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src my_site”). Source: call to eval() or related function blocked by CSP.
handlebars.js:3562
```

**Comment by desultory** [comment]
I was able to get it working by allowing unsafe-eval in the CSP but do not want to keep it like this.  Is there another possible solution?

**Comment by kdslkdsaldsal** [comment]
Sure that the unsafe-eval is the problem? I'm getting the same at e.g. https://demo.owncloud.org/login:

> Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src https://demo.owncloud.org 'unsafe-eval'”). Source: (function injected(eventName, injectedIn....

and still the login and files are working. If i'm checking your owncloud.log i see quite a lot of these:

> "PROPFIND","url":"\/oc\/remote.php\/webdav","message":"Method Not Allowed"}

which shows what you might have some issues with non-allowed WebDAV verbs in your browser. And the files app of oC is exactly using such WebDAV verbs which would exactly explain why you don't see the files overview.

**Comment by desultory** [comment]
all of the issues went away when i allowed unsafe eval and i know it's a good policy to not allow unsafe eval so i'd like to avoid it if possible

**Comment by kdslkdsaldsal** [comment]
So i think we can close this as a duplicate of https://github.com/owncloud/core/issues/11925 for now.

According to that issue eval is currently required and can't be disabled.

**Comment by kdslkdsaldsal** [comment]
Sorry, forgot to mention:

To avoid mixing different logs and info here you could also create a new feature request to show a warning for now if unsafe-eval is not allowed. This could help to know directly whats wrong instead of the need for a user to search for such issues.
